core/src/dbs/session.rs

```diff
@@ -23,12 +23,12 @@ pub struct Session {
 	pub ns: Option<String>,
 	/// The currently selected database
 	pub db: Option<String>,
-	/// The currently selected authentication scope
-	pub sc: Option<String>,
-	/// The current scope authentication token
+	/// The current access method
+	pub ac: Option<String>,
+	/// The current authentication token
 	pub tk: Option<Value>,
-	/// The current scope authentication data
-	pub sd: Option<Value>,
+	/// The current record authentication data
+	pub rd: Option<Value>,
```

Comment:
Even if I believe that the `Level::Record` should be kept as is, I think this proposal still makes sense with regards to exposing a general `data` property that can contain the most specific data about the authenticated actor. Currently, this may only contain the same data as `rd`, but could potentially contain other data about the authenticated actor (e.g. if we support more user-related metadata) that may be relevant for auditing and other purposes that may benefit from direct access to this data whenever working in the context of a session.